data/kobold_press/document.json

```diff
@@ -0,0 +1,245 @@
+[
+    {
+        "title": "Kobold Press OGL Content",
```

Comment:
Is there any chance we can get more specific here, since there's a lot of Kobold Press OGL Content? Maybe we call it "Kobold Press OGL Spells Compilation" or similar?